src/components/form/widgets/code-editor.ts

```diff
@@ -24,6 +24,8 @@ export class CodeEditor extends React.Component {
                 value: value,
                 language: 'json',
                 lineNumbers: true,
+                fold: true,
+                lint: true,
```

Comment:
This is a `feat` not a `fix`… it's adding functionality, not fixing a bug.

Reply:
is it? 
in my opinion, we are fixing the non-existing linting and folding in code editors which are in forms. It's not a functionality for the consumer. They can't turn it off or have a say (as you also mentioned above). The day we make these like overridable pros, then that'd be a `feat`.

Reply:
I also think we are fixing the bug here, it should be a part of the expected behavior of the code editor - not an extra feature